com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/protocol/Requests.java

```diff
@@ -24,7 +24,9 @@
 public class Requests {
 
     public static class ValueFormat {
+        @Deprecated
         public boolean hex;
```

Comment:
`hex` is part of the specification: https://microsoft.github.io/debug-adapter-protocol/specification#Types_ValueFormat, what's the rationale for deprecating this?

Reply:
Because of new functionality we can format values to different formats now, beside hex. And I added new property `ValueFormat#type` instead of `ValueFormat#hex` to support that possibility. I assumed we can delete `ValueFormat#hex` in future with no harm because of no need and marked `hex` as deprecated. Just tell me if I did something wrong and I'll fix it. :)

Reply:
@mylog00 The data types under the folder `com/microsoft/java/debug/core/protocol` are generated from the [DAP](https://microsoft.github.io/debug-adapter-protocol/specification) spec, we can't just change it on debugger side.

It's OK to extend the debug settings to support more format types, but remember to support the old valueFormat.hex as well.

Reply:
When contributing to the DAP, you need to open an issue to discuss the proposal first. Only if the proposal is accepted, then open a PR.

A quick search and found a duplicated issue for more formatting options. https://github.com/microsoft/debug-adapter-protocol/issues/197

Reply:
Got it. Thanks)

Reply:
Hi! If there is no other issue could you approve this PR?)